doc/kyua.1.in

```diff
@@ -62,15 +62,19 @@ simple fix to a library to the system administrator deploying a new
 release on a production machine.
 .Pp
 Kyua is able to execute test programs written with a plethora of testing
-libraries and languages.  The library of choice is ATF, for which Kyua
-was originally designed, but simple, framework-less test programs and
-TAP-compliant test programs can also be executed through Kyua.
+libraries and languages.
+The test program library of choice is ATF, which
+.Nm 's
```

Comment:
It does not work as you thought it works :) It only prints `'s` now.

Reply:
That's unfortunate :/.. I'll restore that hardcoded reference.

Reply:
Try `.Nm Ns \&'s` instead ;)

Reply:
I don't know if the GitHub UI is tricking me, but this doesn't seem resolved.

Reply:
> I don't know if the GitHub UI is tricking me, but this doesn't seem resolved.

I forgot to handle it before, but it's fixed properly now.